rawx/handler_chunk.go

```diff
@@ -245,25 +245,25 @@ func (rr *rawxRequest) uploadChunk() {
 func (rr *rawxRequest) copyChunk() {
 	var err error
 	if rr.chunk, err = retrieveDestinationHeader(&rr.req.Header, rr.rawx, rr.chunkID); err != nil {
-		rr.replyError(err)
+		rr.replyError("copyChunk()", err)
 		return
 	}
 	if err := rr.chunk.retrieveContentFullpathHeader(&rr.req.Header); err != nil {
-		rr.replyError(err)
+		rr.replyError("copyChunk()", err)
 		return
 	}
 
 	// Attempt a LINK in the repository
 	op, err := rr.rawx.repo.link(rr.chunkID, rr.chunk.ChunkID)
 	if err != nil {
-		rr.replyError(err)
+		rr.replyError("copyChunk()", err)
 	} else {
 		// Link created, try to place an xattr
 		err = rr.chunk.saveContentFullpathAttr(op)
 		if err != nil {
 			// Xattr failed, rollback the link itself
 			LogError(msgErrorAction("Setxattr()", rr.reqid, err))
-			rr.replyError(err)
+			rr.replyError("copyChunk()", err)
```

Comment:
Won't we report the same error message twice?

Reply:
Good catch !